WordPress/Classes/ViewRelated/Reader/DetailWebView/ReaderCSS.swift

```diff
@@ -7,32 +7,35 @@ struct ReaderCSS {
 
     private let now: Int = Int(Date().timeIntervalSince1970)
 
-    private let twoDays: Int = 60 * 60 * 24 * 2
+    private let expirationDays: Int = 5
+
+    private var expirationDaysInMs: Int {
```

Comment:
Is the result of this in milliseconds or seconds?

Reply:
Ops! Fixed in 8fabf2babb4f6266509708a6b962bbd82e97d6cc